src/main/java/com/commercetools/sync/inventories/helpers/InventoryEntryIdentifier.java

```diff
@@ -53,34 +55,46 @@ public static InventoryEntryIdentifier of(@Nonnull final InventoryEntry inventor
      * Builds an {@link InventoryEntryIdentifier} instance given an sku and supply channel id.
      *
      * @param inventoryEntrySku the SKU of the inventory entry.
-     * @param inventoryEntryChannelId the channel id of the inventory entry.
+     * @param inventoryEntryChannelKey the channel key of the inventory entry.
      * @return an instance of {@link InventoryEntryIdentifier} for the given entry.
      */
-    public static InventoryEntryIdentifier of(@Nonnull final String inventoryEntrySku,
-                                              @Nullable final String inventoryEntryChannelId) {
-        return new InventoryEntryIdentifier(inventoryEntrySku, inventoryEntryChannelId);
-    }
+    public static InventoryEntryIdentifier of(
+        @Nonnull final String inventoryEntrySku,
+        @Nullable final String inventoryEntryChannelKey) {
 
-    public String getInventoryEntryChannelId() {
-        return inventoryEntryChannelId;
+        return new InventoryEntryIdentifier(inventoryEntrySku, inventoryEntryChannelKey);
     }
 
     public String getInventoryEntrySku() {
         return inventoryEntrySku;
     }
 
+    public String getInventoryEntryChannelKey() {
+        return inventoryEntryChannelKey;
+    }
+
     @Override
-    public boolean equals(final Object obj) {
-        if (obj instanceof  InventoryEntryIdentifier) {
-            final InventoryEntryIdentifier that = (InventoryEntryIdentifier) obj;
-            return Objects.equals(that.getInventoryEntryChannelId(), inventoryEntryChannelId)
-                && Objects.equals(that.getInventoryEntrySku(), inventoryEntrySku);
+    public String toString() {
+        return "{" + "sku='" + inventoryEntrySku + '\'' + ", channelKey='" + inventoryEntryChannelKey + '\'' + '}';
```

Comment:
I think the formatting of this toString method would be better with format method.

Reply:
e422c69